package/MDAnalysis/coordinates/base.py

```diff
@@ -216,6 +224,9 @@ def __init__(self, n_atoms, **kwargs):
         self.has_forces = kwargs.get('forces', False)
 
         self._unitcell = self._init_unitcell()
+        
+        # set up aux namespace for adding auxiliary data
+        self.aux = Namespace()
```

Comment:
+1 for the Namespace solution. (tab-completion plus I'm lazy to type square brackets)